client/app/account/billing/autoRenew/hosting/actions-menu.html

```diff
@@ -34,7 +34,7 @@
                 class="btn btn-link"
                 data-ng-class="{ 'disabled' : !canResiliate(service.subProducts.hosting_web, user) }"
                 data-ng-disabled="!canResiliate(service.subProducts.hosting_web, user)"
-                data-ng-click="canResiliate(service.subProducts.hosting_web, user) && resiliateService(service.subProducts.hosting_web)">
+                data-ng-click="resiliateService(service.subProducts.hosting_web)">
```

Comment:
Some refacto are necessary here:

```diff
     <li data-ng-if="!service.subProducts.hosting_web.renew.deleteAtExpiration">
         <button type="button"
-                class="btn btn-link"
                 data-ng-class="{ 'disabled' : !canResiliate(service.subProducts.hosting_web, user) }"
                 data-ng-disabled="!canResiliate(service.subProducts.hosting_web, user)"
-                data-ng-click="resiliateService(service.subProducts.hosting_web)">
-            <i class="fa fa-trash" aria-hidden="true"></i>
-            <span data-translate="autorenew_service_resiliate_button"></span>
+                data-ng-click="resiliateService(service.subProducts.hosting_web)"
+                data-translate="autorenew_service_resiliate_button">
         </button>
     </li>
```

Would be great to spread this to all actions (`<li>`).